webpack.config.js

```diff
@@ -1,8 +1,8 @@
 /*eslint-env node*/
 /*eslint import/no-nodejs-modules:0 */
 const fs = require('fs');
-
 const path = require('path');
+
```

Comment:
the git hooks seem to have done this after merging with my old branch from master, it does seem to be correct?